proto/osmosis/tokenfactory/v1beta1/tx.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+package osmosis.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v7/x/tokenfactory/types";
+
+// Msg defines the Msg service.
+service Msg {
+  rpc CreateDenom(MsgCreateDenom) returns (MsgCreateDenomResponse);
+  rpc Mint(MsgMint) returns (MsgMintResponse);
+  rpc Burn(MsgBurn) returns (MsgBurnResponse);
+  // ForceTransfer is deactivated for now because we need to think through edge
+  // cases rpc ForceTransfer(MsgForceTransfer) returns
+  // (MsgForceTransferResponse);
+  rpc ChangeAdmin(MsgChangeAdmin) returns (MsgChangeAdminResponse);
+}
+
+// ===================== MsgCreateDenom
+message MsgCreateDenom {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  string nonce = 2 [ (gogoproto.moretags) = "yaml:\"nonce\"" ];
+}
+
+message MsgCreateDenomResponse {
+  string new_token_denom = 1
+      [ (gogoproto.moretags) = "yaml:\"new_token_denom\"" ];
+}
+
+// ===================== MsgMint
+message MsgMint {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  cosmos.base.v1beta1.Coin amount = 2 [
+    (gogoproto.moretags) = "yaml:\"amount\"",
+    (gogoproto.nullable) = false
+  ];
+  string mintToAddress = 3
+      [ (gogoproto.moretags) = "yaml:\"mint_to_address\"" ];
+}
+
+message MsgMintResponse {}
+
+// ===================== MsgBurn
+message MsgBurn {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  cosmos.base.v1beta1.Coin amount = 2 [
+    (gogoproto.moretags) = "yaml:\"amount\"",
+    (gogoproto.nullable) = false
+  ];
+  string burnFromAddress = 3
+      [ (gogoproto.moretags) = "yaml:\"burn_from_address\"" ];
+}
+
+message MsgBurnResponse {}
+
+// // ===================== MsgForceTransfer
+// message MsgForceTransfer {
+//   string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+//   cosmos.base.v1beta1.Coin amount = 2 [
+//     (gogoproto.moretags) = "yaml:\"amount\"",
+//     (gogoproto.nullable) = false
+//   ];
+//   string transferFromAddress = 3
+//       [ (gogoproto.moretags) = "yaml:\"transfer_from_address\"" ];
+//   string transferToAddress = 4
+//       [ (gogoproto.moretags) = "yaml:\"transfer_to_address\"" ];
+// }
+
+// message MsgForceTransferResponse {}
```

Comment:
Do we need this commented out?

Reply:
Wanted to deactivate this message for now, but we will eventually add it back in in the future

Reply:
Should we track this in an issue?